apis/sgv2-graphqlapi/pom.xml

```diff
@@ -10,6 +10,7 @@
   <properties>
     <quarkus.container-image.group>stargateio</quarkus.container-image.group>
     <quarkus.container-image.name>graphqlapi</quarkus.container-image.name>
+    <quarkus.container-image.tag>v${project.version}</quarkus.container-image.tag>
```

Comment:
Our existing code prepends a "v" before the version, whereas Quarkus doesn't. I don't mind either way but if we drop it we'll have to adapt the coordinator and docker-compose scripts.

Reply:
Can you add this to docs as well.. the release process correctly sets the tag with `v`, so this would be only for local builds and keeping that docker-compose stuff working